src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -94,8 +94,9 @@ public void should_idempotently_create_case_from_valid_exception_record() throws
         assertThat(bulkScanCaseReference.equals(String.valueOf(exceptionRecord.getId())));
 
         await("Case is ingested")
-            .atMost(2, TimeUnit.SECONDS)
-            .pollDelay(1, TimeUnit.SECONDS)
+            .atMost(30, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .pollInterval(1, TimeUnit.SECONDS)
```

Comment:
do we need this if we have poll delay ?

Reply:
I removed it. The pollInterval should be enough